config/afmd/globalConfig.sh

```diff
@@ -0,0 +1,140 @@
+# Start by exporting the shared drmaa libaries to the LD_LIBRARY_PATH
```

Comment:
Just dropping a comment here to let you know that my intention is that in the future this file should be removed and all config should instead be moved into the xml files, to not spread it all over the place. But since some workflows still rely on this I haven't gotten rid of it yet.